code/game/turfs/open.dm

```diff
@@ -102,6 +102,50 @@
 	icon_state = "seadeep"
 	can_bloody = FALSE
 
+//Prison Ocean
+
+/turf/open/opensea
+	name = "open space"
+	desc = "You feel dizzy just looking at the waves far below you."
+	icon = 'icons/turf/ground_map.dmi'
+	icon_state = "opensea"
+	can_bloody = FALSE
+
+var/list/canfall = typecacheof(list(
+					/mob/living,
+					/obj/vehicle,
+					/obj/machinery,
+					/obj/item,
+					/obj/structure))
+
+var/list/nofall = typecacheof(list(
+					/obj/structure/lattice,
+					/obj/structure/monorail))//just for specific things that should be able to float
```

Comment:
Either indent these or turn them into proper global lists.

Reply:
GLOBAL_LIST_INIT is the proper macro I think. No idea if ocean turfs are common, if they are turn these into globals.

Reply:
They are common, I'll do that